src/main/java/org/hyperskill/hstest/dev/stage/BaseStageTest.java

```diff
@@ -28,7 +28,7 @@
     private Class userClass;
     private final Object testedObject;
     private final Method testedMethod;
-    protected boolean isTestingMain = false; // TODO potentially it is possible to make it final
+    protected final boolean isTestingMain;
```

Comment:
Now this field shouldn't be acceptable in the inherited classes, so it can be `private`

Reply:
Done.